server/pages.js

```diff
@@ -9,6 +9,7 @@ const routes = require('next-routes');
 const pages = routes()
   .add('home', '/', 'index')
   .add('become-a-sponsor', '/become-a-sponsor', 'become-a-sponsor')
+  .add('become-a-host', '/(become-a-sponsor|become-a-fiscal-host)', 'become-a-host')
```

Comment:
I don't get why `become-a-sponsor` is here?

Reply:
It was a mistake, thanks.